px-chart.js

```diff
@@ -1154,7 +1154,10 @@ Polymer({
               color: "rgb(0,0,0)"
             },
             x: 3,
-            y: 12
+            y: 12,
```

Comment:
Would you mind adding a property that isn't bindable (like the lot of the properties on this component) that can toggle the default formatter with this one?